_src/pldoc/static/js/color-contrast.js

```diff
@@ -130,5 +132,4 @@ $(function() {
     };
 
     AccessibilityColorContrast.init();
-
 });
```

Comment:
I don't remember reviewing this file. There are some examples of 
```
var that = this;
```
which should be cleaned up and some variables not getting declared at the tops of functions.